config-bootstrapper/src/main/java/org/hypertrace/core/bootstrapper/BootstrapContext.java

```diff
@@ -18,26 +16,32 @@
 
   private final AttributeServiceClient attributesServiceClient;
   private final EntityTypeServiceClient entityTypeServiceClient;
+  private final org.hypertrace.entity.type.client.EntityTypeServiceClient entityTypeServiceClientV2;
 
-  private final ManagedChannel attributeServiceChannel;
-  private final ManagedChannel entityServiceChannel;
   private final List<ManagedChannel> managedChannels;
 
   private BootstrapContext(
       AttributeServiceClientConfig attributeServiceClientConfig,
       EntityServiceClientConfig entityServiceClientConfig) {
 
-    this.attributeServiceChannel = ManagedChannelBuilder.forAddress(
+    ManagedChannel attributeServiceChannel = ManagedChannelBuilder.forAddress(
         attributeServiceClientConfig.getHost(), attributeServiceClientConfig.getPort())
         .usePlaintext().build();
     this.attributesServiceClient = new AttributeServiceClient(attributeServiceChannel);
 
-    this.entityServiceChannel = ManagedChannelBuilder.forAddress(
+    ManagedChannel entityServiceChannel = ManagedChannelBuilder.forAddress(
         entityServiceClientConfig.getHost(), entityServiceClientConfig.getPort())
         .usePlaintext().build();
     this.entityTypeServiceClient = new EntityTypeServiceClient(entityServiceChannel);
 
-    managedChannels = List.of(attributeServiceChannel, entityServiceChannel);
+    ManagedChannel entityTypeServiceChannel = ManagedChannelBuilder.forAddress(
```

Comment:
This is the same channel as the one above - no need to create a second channel for the same host.

Reply:
ahh.. thanks. fixed.